testsuite/features/build_validation/reposync/srv_sync_all_products.feature

```diff
@@ -767,6 +767,38 @@ Feature: Synchronize products in the products page of the Setup Wizard
     When I use spacewalk-common-channel to add all "debian-12" channels with arch "amd64-deb"
     And I wait until all synchronized channels for "debian-12" have finished
 
+@cloud
+@proxy
+  Scenario: Add Manager Proxy 5.0 Public Cloud channels
+    When I add "sle-module-public-cloud15-sp6-pool-x86_64-proxy-5.0" channel
+    And I wait until the channel "sle-module-public-cloud15-sp6-pool-x86_64-proxy-5.0" has been synced
+    And I add "sle-module-public-cloud15-sp6-updates-x86_64-proxy-5.0" channel
+    And I wait until the channel "sle-module-public-cloud15-sp6-updates-x86_64-proxy-5.0" has been synced
+
+@uyuni
+@proxy
+  Scenario: Add Uyuni Leap 15.5 Proxy, including Uyuni Client Tools
+    When I use spacewalk-common-channel to add all "leap15.5" channels with arch "x86_64"
+    And I wait until all synchronized channels for "uyuni-proxy" have finished
+
+@susemanager
+@proxy
+  Scenario: Add SUSE Linux Enterprise Micro 5.5
```

Comment:
```suggestion
  Scenario: Add SUSE Linux Enterprise Micro 5.5 Proxy
```

Reply:
eh it's not exactly, it's the base product, but it's also used for proxy. We can change the scenario name to be unique though, yes

Reply:
Just to keep the consistency with the previous scenario that we're replacing - only because I was looking at the scenario and not understanding why it had the `@proxy` tag

Reply:
Changed in the new commit